net/http/src/THttpServer.cxx

```diff
@@ -1155,8 +1155,9 @@ Bool_t THttpServer::ExecuteWS(std::shared_ptr<THttpCallArg> &arg, Bool_t externa
          TUrl url;
          url.SetOptions(arg->fQuery);
          url.ParseOptions();
-         Int_t connid = url.GetIntValueFromOptions("connection");
-         arg->SetWSId((UInt_t)connid);
+         const char *connid = url.GetValueFromOptions("connection");
+         if (connid)
+            arg->SetWSId(std::stoul(connid));
```

Comment:
Note that `std::stoul` return a `long`, which is still 32 bit on Win64 (if it's relevant at all)

Reply:
It should be `ULong_t`. Is it correct?

Reply:
As I can see, on win64 `sizeof(ULong_t)` is also 4 bytes.
In principle, original problem is `TString::Hash`, which returns `UInt_t`.
I guess, `unsigned long` always should be sufficient for `UInt_t`.